torch/onnx/symbolic_helper.py

```diff
@@ -226,17 +226,18 @@ def _sort_helper(g, input, dim, decending=True, out=None):
 def _topk_helper(g, input, k, dim, largest=True, sorted=False, out=None):
     if out is not None:
         _unimplemented("TopK", "Out parameter is not supported")
+    if not largest:
+        _unimplemented("TopK", "Ascending is not supported")
+    k = _maybe_get_const(k, 'i')
+    if not _is_value(k):
+        k = g.op("Constant", value_t=torch.tensor(k, dtype=torch.int64))
     if _export_onnx_opset_version <= 10:
-        if not largest:
-            _unimplemented("TopK", "Ascending is not supported")
-            return g.op("TopK", input, k_i=k, axis_i=dim, outputs=2)
-        k = _maybe_get_const(k, 'i')
-        if not _is_value(k):
-            k = g.op("Constant", value_t=torch.tensor(k, dtype=torch.int64))
         from torch.onnx.symbolic_opset9 import unsqueeze
         k = unsqueeze(g, k, 0)
```

Comment:
is `k` required to be 1-D tensor for ONNX::TopK? Reshaping to shape [1] is safer because unsqueeze assumes previous `k` is always scalar.

Reply:
Yes it is. I updated to reshape. Thanks.